<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>loading_widget</class>
    <widget class="QWidget" name="loading_widget">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>247</width>
                <height>232</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <property name="styleSheet">
            <string notr="true">background-color: rgb(77, 80, 87);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <layout class="QHBoxLayout" name="loading_layout">
                    <item>
                        <spacer name="loading_left_spacer">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QLabel" name="loading_movie_label">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="minimumSize">
                                <size>
                                    <width>150</width>
                                    <height>150</height>
                                </size>
                            </property>
                            <property name="text">
                                <string/>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="loading_right_spacer">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QLabel" name="loading_label">
                    <property name="text">
                        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span
                            style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;Looking for text
                            matches...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                        </string>
                    </property>
                </widget>
            </item>
            <item>
                <spacer name="loading_label_bottom_spacer">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>24</height>
                        </size>
                    </property>
                </spacer>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
